src/styles/theme.ts

```diff
@@ -13,6 +13,10 @@ const theme = createTheme({
     error: {
       main: red.A400,
     },
+    text: {
+      primary: '#ffffff',
+      secondary: '#007bff',
```

Comment:
@evanjscallan I didn't catch this when I reviewed before merging, but if you set this here it actually overrides the default text color everywhere (sets the primary color on the `<body>`). I removed it in #57, so the button text won't be white anymore. We can figure out how to add it back in your newest PR

Reply:
Got it! I'll see what if there are any tertiary assignments from theme that can remain independent and work from there.